frontend/src/components/service-cards.tsx

```diff
@@ -0,0 +1,231 @@
+import {
+  Container,
+  GitBranchIcon,
+  Github,
+  HardDrive,
+  LinkIcon,
+  Tag
+} from "lucide-react";
+import { cn } from "~/lib/utils";
+import {
+  Card,
+  CardContent,
+  CardFooter,
+  CardHeader,
+  CardTitle
+} from "./ui/card";
+import { Separator } from "./ui/separator";
+import {
+  Tooltip,
+  TooltipContent,
+  TooltipProvider,
+  TooltipTrigger
+} from "./ui/tooltip";
+
+type DockerServiceCardProps = {
+  name: string;
+  image: string;
+  url?: string;
+  tag: string;
+  updatedAt: string;
+  volumeNumber?: number;
+  status: "healthy" | "unhealthy" | "sleeping" | "undeployed";
+};
+
+export function DockerServiceCard({
+  name,
+  image,
+  url,
+  tag,
+  updatedAt,
+  volumeNumber = 0,
+  status
+}: DockerServiceCardProps) {
+  return (
+    <Card className="rounded-2xl flex flex-col h-[220px] bg-toggle relative">
+      <TooltipProvider>
+        <Tooltip delayDuration={0}>
+          <TooltipTrigger asChild>
+            <span className="absolute cursor-pointer flex h-4 w-4 -top-2 -right-2">
+              {status !== "undeployed" && (
+                <span
+                  className={cn(
+                    "animate-ping absolute inline-flex h-full w-full rounded-full  opacity-75",
+                    {
+                      "bg-green-400": status === "healthy",
+                      "bg-red-400": status === "unhealthy",
+                      "bg-secondary": status === "sleeping"
+                    }
+                  )}
+                />
+              )}
+
+              <span
+                className={cn(
+                  "relative inline-flex rounded-full h-4 w-4 bg-green-500",
+                  {
+                    "bg-green-500": status === "healthy",
+                    "bg-red-500": status === "unhealthy",
+                    "bg-secondary": status === "sleeping",
+                    "bg-gray-400": status === "undeployed"
+                  }
+                )}
+              ></span>
+            </span>
+          </TooltipTrigger>
+          <TooltipContent>
+            <div>
+              {status === "healthy" && "✅ "}
+              {status === "sleeping" && "💤 "}
+              {status === "unhealthy" && "❌ "}
+              {status === "undeployed" && "⏳ "}
+              {status}
```

Comment:
The text should be `not deployed yet` if the status is `undeployed`

Reply:
didn't understand what you mean